src/subscription/azext_subscription/_params.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.parameters import get_enum_type
+
+
+# pylint: disable=line-too-long
+def load_arguments(self, _):
+    with self.argument_context('account subscriptiondefinition create') as c:
```

Comment:
From the help file, looks like the command is `subscriptiondefinition create`

Reply:
I've updated the help file to use "account subscription-definition create".